docs/crio.conf.5.md

```diff
@@ -116,6 +116,9 @@ The `crio.runtime` table contains settings pertaining to the OCI runtime used an
 **no_pivot**=false
   If true, the runtime will not use `pivot_root`, but instead use `MS_MOVE`.
 
+**container-min-memory**="12MiB"
```

Comment:
This might be out of place for `crio.conf`. Looks like it should be included in the CLI documentation.